openmls/src/storage.rs

```diff
@@ -0,0 +1,64 @@
+#[derive(Debug, Clone, Default)]
+pub struct OpenMlsTypes;
+
+use openmls_traits::storage::*;
+
+use crate::ciphersuite::hash_ref::ProposalRef;
+use crate::group::GroupContext;
+use crate::group::GroupId;
+use crate::group::InterimTranscriptHash;
+use crate::group::QueuedProposal;
+use crate::messages::ConfirmationTag;
+use crate::treesync::TreeSync;
+
+impl Entity<1> for QueuedProposal {}
```

Comment:
Let's not use literals here for the versions but constants to make it harder to mess up.